documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -16,6 +16,7 @@ and the `bmc` `address` parameter for the `install-config.yaml` file.
 provisioning network. ({product-title} 4.4 only)
 | `hosts` |  | Details about bare metal hosts to use to build the cluster.
 | `defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration.
+| `disableCertificateVerification` | | This parameter is needed when we use `redfish` or `redfish-virtualmedia` to manage BMC addresses. The value should be `True` if you are using self-signed certificate for BMC addresses or `False` if you have a Certificate Authority. 
```

Comment:
Lets leave the or False part out. Lets put False in the default column so it should look like

| `disableCertificateVerification` | False | This parameter is needed when we use `redfish` or `redfish-virtualmedia` to manage BMC addresses. The value should be `True` if you are using self-signed certificate for BMC addresses.

Reply:
Yeah, sounds good.

Reply:
Looks good, squash down to 1 commit?

Reply:
yeah, sure squashing it now.